### DISCUSSION TITLE
Applying a sort effect on a Project -> View prevents moving them between columns

### DISCUSSION THREAD

**Discussion by ividu** [community/community#44217] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

Applying a `sort` effect to a View on a given Project will disable drag&drop on all the items/issues. While this is expected within a column, as the sorting is automatic, users should be able to move items to a new column still. 

The sort should only block movement on the vertical axis, not horizontally between columns. 

_Issue refers to the new Projects, not the legacy ones._

**Comment by rileybroughten** [comment, marked as answer]
👋 Hello @ividu! We recently released an improvement that allows you to sort by field values on the board layout, while still being able to move items between columns and maintaining the sorted order - check out our [changelog announcement](https://github.blog/changelog/2023-01-26-github-projects-january-26th-update/#sorting-by-field-values-on-the-board-layout) and [docs](https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/customizing-the-board-layout#sorting-by-field-values) which highlight this! Let me know if you have any questions or feedback 👍
